synapse/lib/scrape.py

```diff
@@ -70,6 +70,14 @@ def fqdn_check(match: regex.Match):
             return None, {}
     return valu, {}
 
+def cve_check(match: regex.Match):
+    mnfo = match.groupdict()
+    valu = mnfo.get('valu')  # type: str
+    cbfo = {}
+
+    valu = valu.replace('\u2013', '-')
```

Comment:
did you consider having this be a norm step for it:sec:cve?

Reply:
Yea, you could manage that via a `replace` option in the type decl so that copy/pasted values into storm queries will also get fixed up. Good call @mikemoritz